query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/Params.java

```diff
@@ -15,21 +16,21 @@
   private final Map<Integer, String> stringParams;
   private final Map<Integer, Float> floatParams;
   private final Map<Integer, Double> doubleParams;
-  private final Map<Integer, String> bytesStringParams;
+  private final Map<Integer, ByteString> byteStringParams;
```

Comment:
Leaving the bytes params as bytes till the last minute will let us to do the correct transformation at the end.